mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/JPAMailboxManager.java

```diff
@@ -45,6 +45,7 @@
 
     public static final EnumSet<MailboxCapabilities> MAILBOX_CAPABILITIES = EnumSet.of(MailboxCapabilities.UserFlag,
         MailboxCapabilities.Namespace,
+        MailboxCapabilities.Move,
```

Comment:
once hbase and jrc will be removed, it seems that will be able to remove this capability, as every backends are implementing it